.github/workflows/release.yaml

```diff
@@ -59,6 +59,9 @@ jobs:
 
           # create tarball
           cd ./output && tar -cvzf manifests.tar.gz $files
+      - name: Generate install manifest
+        run: |
+          kustomize build ./manifests/install > ./output/install.yaml
```

Comment:
Having looked more into how GoReleaser does things, I would maybe change the `./output` folder to `./dist` so that we have all things related to the release build in one place.

Another option would be to make it part of the `GoReleaser` process, as it can work with custom commands. That would make it reproducible and a one-stepper.

Reply:
I think you'd need to install `kustomize` inside the GoReleaser runner for that work.

Maybe `dist/` should be added to `.gitignore` as well? I see there is one entry for `output/` already.

Reply:
Mmm, that's actually a valid concern. Does the action have access to the files generated prior?

Reply:
I removed the GH Actions!